lib/contracts/voteInOrganizationScheme.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+const dopts = require("default-options");
```

Comment:
by the way, why not start using typescript for new wrappers?

Reply:
I created an issue  just for that.  I can't be sure there won't be issues with tools and packaging, so want to be able to concentrate on that.